src/sound/Sound.cpp

```diff
@@ -552,7 +552,7 @@ namespace Sound {
 
 	class LoadSoundJob : public Job {
 	public:
-		LoadSoundJob(std::string directory, bool isMusic) :
+		LoadSoundJob(const std::string &directory, const bool isMusic) :
```

Comment:
Another const-primitive-type here - the value isn't passed by reference and doesn't need to be made const to prevent accidental writes from leaking.